src/essentia/essentiamath.h

```diff
@@ -1200,6 +1200,142 @@ std::vector<std::vector<T> > pairwiseDistance(const std::vector<std::vector<T> >
   return pdist;
 }
 
+/**
+ * Sets `squeezeShape`, `summerizerShape`, `broadcastShape` to perform operations 
+ * on a Tensor with the shape of `tesor` along the `axis` dimension.
+ */
+template <typename T>
+void tensorGeometricalInfo(const Tensor<T>& tensor, int& axis,
```

Comment:
We can consider simplify the name: `Geometrical` -> `Geometric`. Also for consistency with other names (e.g., we have the `GeometricMean` algorithm)

Reply:
I used `geometrical`  for consistency with  Eigen [Geometrical operations ](https://eigen.tuxfamily.org/dox/unsupported/eigen_tensors.html#autotoc_md95), but I do not have a preference for one name or another.

Reply:
Then, this is fine